src/ActionModal/utils/ActionManager.js

```diff
@@ -237,13 +237,13 @@ export default class ActionManager {
 
 function convertCurrencyData(amounts) {
   return amounts.map(({ amount, denom }) => ({
-    amount: toMicroAtomString(amount),
+    amount: toGasMilliUnitString(amount),
     denom: toMicroDenom(denom)
   }))
 }
 
-function toMicroAtomString(amount) {
-  return String(uatoms(amount))
+function toGasMilliUnitString(amount) {
```

Comment:
this will probably need to take in a denom to determin the currency conversion. we could serve the currency conversions from the API

Reply:
you mean to show all values in fiat right? I think this would make another PR though

Reply:
no, to convert different tokens with different milli unit precision

Reply:
but yeah let's do it another PR